news/2018-02-25-project-loved-week-of-february-25th.md

```diff
@@ -0,0 +1,287 @@
+
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+One of the most popular unranked maps around, [Raikozen](https://osu.ppy.sh/users/3827077)'s [Warriors](https://osu.ppy.sh/beatmapsets/431135) became incredibly popular amongst top players following a version made by [Axarious](https://osu.ppy.sh/users/2614511) for OWC 2015 finals, though sitting at 8.32\*, it is exceptionally harder. Large full-screen jumps litter the screen in this high-energy pump-up mantra that hails over 475 favorites.
+
+You aren't ready...
+
+[![](/wiki/shared/news/2018-02-25-project-loved-week-of-february-25th/osu/bandiria-travellers.jpg)](https://osu.ppy.sh/community/forums/topics/708903)
+
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+If anyone remembers watching livestreams from old top player [thelewa](https://osu.ppy.sh/users/475021), you may recognize this map as the song he used to end every stream. [jesse1412](https://osu.ppy.sh/users/230116)'s near perfect emulation of old-style [Starrodkirby86](https://osu.ppy.sh/users/410) and [mtmcl](https://osu.ppy.sh/users/5960) mapping is seen within throughout [Bandiria Travellers](https://osu.ppy.sh/beatmapsets/51391). Low AR, linear patterns, even the artist choice perfectly resembles an era of mapping very dear to those who enjoy old-style mapping. Leave the storyboard on, and shed a tear along with Susumu Hirasawa himself for this masterpiece of a map.
+
+[![](/wiki/shared/news/2018-02-25-project-loved-week-of-february-25th/osu/narcissus-at-oasis.jpg)](https://osu.ppy.sh/community/forums/topics/708904)
+
+*written by [Kaifin](https://osu.ppy.sh/users/2596942)*
+
+Next on the list is a map you may not have tried which has recieved quite a few submissions to the loved spreadsheet since its inception! [Narcissus at Oasis \(Freezer Mix\)](https://osu.ppy.sh/beatmapsets/614615) by [EijiKuinbii](https://osu.ppy.sh/users/6270995) is a gigantic 3 mode set featuring several guest difficulties in different mapping styles! With diffs ranging from low to high intensity, there's something here for everyone to enjoy.
+
+## <a name="osutaiko" id="osutaiko"></a>osu!taiko
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/tstFskmG0MM?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+---
+
+[![](/wiki/shared/news/2018-02-25-project-loved-week-of-february-25th/taiko/los-los-los.jpg)](https://osu.ppy.sh/community/forums/topics/708912)
+
+*written by [Nwolf](https://osu.ppy.sh/users/1910766)*
+
+A bit of a normal break from all the insanity, there isn't much to actually say about [Los! Los! Los!](https://osu.ppy.sh/beatmapsets/603361) other than it being an outstanding anime map in the sea of them. Long, energetic bursts of energy characterize this map as a force, with very dense sections of 1/4 triplets. Los! captures our hearts in full.
```

Comment:
A bit from?

Reply:
I think it's correct as-is

it's "a break from ..." with some extra stuff